common/djangoapps/enrollment/serializers.py

```diff
@@ -99,3 +103,45 @@ class ModeSerializer(serializers.Serializer):
     description = serializers.CharField()
     sku = serializers.CharField()
     bulk_sku = serializers.CharField()
+
+
+# Edraak: adding custom serializer for enrollments
+class EdraakCourseEnrollmentSerializer(CourseEnrollmentSerializer):
+    edraak_course_details = EdraakCourseSerializer(source="course_overview")
+    is_certificate_allowed = serializers.SerializerMethodField()
+    specialization_slug = serializers.SerializerMethodField()
+
+    def get_is_certificate_allowed(self, obj):
+        from edraak_certificates.utils import is_certificate_allowed
+
+        request = self.context.get('request')
```

Comment:
```suggestion
        request = get_request_or_stub()
```

Please import it locally instead.